graph/telemetry/istio/istio.go

```diff
@@ -388,15 +386,14 @@ func addTraffic(trafficMap graph.TrafficMap, metric string, inject bool, val flo
 // addEdgeTraffic uses edgeTSHash that the metric information has not been applied to the edge. Returns true
 // if the the metric information is applied, false if it determined to be a duplicate.
 func addEdgeTraffic(trafficMap graph.TrafficMap, val float64, protocol, code, flags, host string, source, dest *graph.Node, edgeTSHash string, o graph.TelemetryOptions) bool {
-
 	var edge *graph.Edge
 	for _, e := range source.Edges {
 		if dest.ID == e.Dest.ID && e.Metadata[graph.ProtocolKey] == protocol {
 			edge = e
 			break
 		}
 	}
-	if nil == edge {
+	if edge == nil {
```

Comment:
funny detail to change :-)

Reply:
Just a personal nit :)

Reply:
Always got in the habit to put constants first to avoid mistaken assignment as opposed to equality.  The habit has carried over even when it's not necessary.  But there was/is a reason.

Reply:
Oh well I'll just revert then